huaweicloud/services/acceptance/bms/resource_huaweicloud_bms_instance_test.go

```diff
@@ -53,8 +54,8 @@ func TestAccBmsInstance_basic(t *testing.T) {
 }
 
 func testAccCheckBmsInstanceDestroy(s *terraform.State) error {
-	config := acceptance.TestAccProvider.Meta().(*config.Config)
-	bmsClient, err := config.BmsV1Client(acceptance.HW_REGION_NAME)
+	cfg := acceptance.TestAccProvider.Meta().(*config.Config)
+	bmsClient, err := cfg.BmsV1Client(acceptance.HW_REGION_NAME)
 	if err != nil {
 		return fmt.Errorf("Error creating HuaweiCloud bms client: %s", err)
```

Comment:
please check other error messages
```suggestion
		return fmt.Errorf("error creating bms client: %s", err)
```

Reply:
fixed